agentforce/1.0/modules/ROOT/pages/agentforce-connector-studio.adoc

```diff
@@ -0,0 +1,148 @@
+= Using Anypoint Studio to Configure MuleSoft Chain AI Connector 1.0 - Mule 4
+:exchange-search-name: mulesoft ai chain
+:connector-name: Agentforce
+:lc-connector-name: agentforce
+:studio-connector-name: Agentforce Connector
+
+include::connectors::partial$t-studio-partial.adoc[tags=topic-intro]
+
+
+[[create-mule-project]]
+== Create a Mule Project
+
+include::connectors::partial$t-studio-partial.adoc[tags=create-project-intro]
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+Add Agentforce Connector to your Mule project to automatically populate the XML code with the connector's namespace and schema location and add the required dependencies to the project's `pom.xml` file:
+
+. In *Mule Palette*, click *(X) Search in Exchange*.
+. In *Add Dependencies to Project*, type `mulesoft ai chain` in the search field.
+. Click *Agentforce Connector* in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+[[configure-source]]
+== Configure a Source
+
+include::connectors::partial$t-studio-partial.adoc[tags=configure-source-intro]
+You can configure one of these sources to use with MuleSoft Chain AI Connector:
+
+include::connectors::partial$t-studio-partial.adoc[tags=listener-scheduler-descriptions]
+
+include::connectors::partial$t-studio-partial.adoc[tags=listener-steps]
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you are specifying an action for that connector to perform.
+
+To add an operation for Agentforce Connector, follow these steps:
+
+. In *Mule Palette*, select *Agentforce Connector* and then select the operation to add.
++
+image:agentforce-palette-operations.png["Mule palette in Studio with Agentforce selected and all of the connector's operations listed"]
+. Drag the operation onto the Studio canvas, next to the source.
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+When you configure a connector, configure a global element that all instances of that connector in the app can use. Configuring a global element requires you to provide the authentication credentials that the connector requires to access the target system.
+
+include::connectors::partial$t-studio-partial.adoc[tags=property-placeholders]
+
+To configure the global element for Agentforce Connector, follow these steps:
+
+. Select the operation in the Studio canvas.
+. On the properties screen for the operation, click the Add (*+*) icon  to access the global element configuration fields.
```

Comment:
This configuration steps are still yet to be updated right? These appear to be c&ped from the original Mule AI Chain connector